### PR TITLE
Add multiple ssh keys

### PR DESCRIPTION
**Pull request by alvarosimon**
Include a new subsection to explain how to generate different ssh keys from different computers.

### CODE REVIEW

File: intro-HPC/ch_account.tex [hunk 1]
```diff
@@ -433,6 +433,24 @@ \subsection{Welcome e-mail}
 Now, you can start using the \hpc. You can always look up your vsc id later by visiting\\*
 \url{https://account.vscentrum.be}.
 
+\subsection{Adding multiple keys (optional)}
```
Comment (boegel) [inline intro-HPC/ch_account.tex]: be more specific: `keys` -> `SSH public keys`

File: intro-HPC/ch_account.tex [hunk 2]
```diff
@@ -433,6 +433,24 @@ \subsection{Welcome e-mail}
...
+\label{sec:adding-multiple-keys}
+In case you are connecting from different computers to the login nodes, it is advised to use separate keys to do so.
```
Comment (boegel) [inline intro-HPC/ch_account.tex]: `keys` -> `SSH public keys`

File: intro-HPC/ch_account.tex [hunk 3]
```diff
@@ -433,6 +433,24 @@ \subsection{Welcome e-mail}
...
+You should follow these steps.
+
+\begin{enumerate}
+\ifmacORlinux
+ \item Create a new public/private key pair from the new computer. Repeat the process described in section~\ref{sec:generate-key-pair-with-openssh}.
```
Comment (boegel) [inline intro-HPC/ch_account.tex]: `key pair` -> `SSH key pair`

File: intro-HPC/ch_account.tex [hunk 4]
```diff
@@ -433,6 +433,24 @@ \subsection{Welcome e-mail}
...
+\fi %macORlinux
+\ifwindows
+ \item Create a new public/private key pair from Putty. Repeat the process described in section~\ref{sec:generate-key-pair}.
+\fi %ifwindows
+ \item Go to \url{https://account.vscentrum.be/django/vo/edit}
+ \item Upload the new public key using the \strong{Add public key} section.
```
Comment (boegel) [inline intro-HPC/ch_account.tex]: `new public key` -> `new SSH public key`

File: intro-HPC/ch_account.tex [hunk 5]
```diff
@@ -433,6 +433,24 @@ \subsection{Welcome e-mail}
...
+ \item (optional) If you lost your key, you can delete the old key on the same page.
```
Comment (boegel) [inline intro-HPC/ch_account.tex]: mention that there always needs to be at least 1 SSH public key registered (so you can't delete and then add when you want to replace the existing key, you need to add then delete (the right one))

File: intro-HPC/ch_account.tex [hunk 6]
```diff
@@ -433,6 +433,24 @@ \subsection{Welcome e-mail}
...
+ \item After 15 minutes the changs should have taken effect.
```
Comment (boegel) [inline intro-HPC/ch_account.tex]: Make this 15-30 minutes, and this may be site-specific.

Maybe reword to:

```
Take into account that it will take some time before the new SSH public key is active in your account on the system; waiting for 15-30 minutes should be sufficient.

**Comment by alvarosimon** [comment]
@boegel remarks addressed

**Comment by boegel** [comment]
@alvarosimon You'll need to run `check-version.sh` again to make Travis happy...

**Comment by alvarosimon** [comment]
@boegel ah.. I always forget this  :sweat_smile: done!
